src/psij/web/js/customization.js

```diff
@@ -29,22 +29,22 @@ var CONF = {
                         scheduler: "SLURM"
                     },
                     "illyad.hidden.uoregon.edu": {
-                        scheduler: "Cron"
+                        scheduler: "SLURM"
```

Comment:
Those machines don't have a job scheduler. I think only axis at uoregon has a scheduler and it's slurm

Reply:
https://github.com/ExaWorks/psij-testing-service/issues/41